src/Foundation/InstalledApp.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Foundation;
+
+use Flarum\Database\Console\GenerateMigrationCommand;
+use Flarum\Database\Console\MigrateCommand;
+use Flarum\Database\Console\ResetCommand;
+use Flarum\Foundation\Console\CacheClearCommand;
+use Flarum\Foundation\Console\InfoCommand;
+use Flarum\Http\Middleware\DispatchRoute;
+use Zend\Diactoros\Response\HtmlResponse;
+use Zend\Stratigility\MiddlewarePipe;
+use function Zend\Stratigility\middleware;
+use function Zend\Stratigility\path;
+
+class InstalledApp implements AppInterface
+{
+    /**
+     * @var Application
+     */
+    protected $laravel;
+
+    /**
+     * @var array
+     */
+    protected $config;
+
+    public function __construct(Application $laravel, array $config)
+    {
+        $this->laravel = $laravel;
```

Comment:
`$laravel` isn't really the right variable name to use here. I understand wanting to use it, but it would highly confuse people. Also, everywhere else in Flarum we have assigned Application to the variable name `$app` why move away from that here? Just because the class is called `InstalledApp`?